lib/mvos/ui/widgets/create_account_form.dart

```diff
@@ -0,0 +1,255 @@
+                  controller: _passwordController,
+                  validator: (value) {
+                   if (value.length < 6) {
+                     return 'Please enter more text';
+                   }  return null;
+                  },
+                  keyboardType: TextInputType.phone,
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    icon: Icon(Icons.keyboard),
+                    labelText: 'Password',
+                    border: OutlineInputBorder(
+                      borderRadius: BorderRadius.all(
+                        Radius.circular(5.0),
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(
+                height: 20,
+              ),
+              Container(height: 50,
+                child: TextFormField(
+                  focusNode: _repeatPasswordFocusNode,
+                  textInputAction: TextInputAction.go,
+                  onEditingComplete: () => _codeFocusNode.requestFocus(),
+                  controller: _repeatPasswordController,
+                  validator: (value) {
+                  if (value != _passwordController.text) {
+                    return 'Password is not matching';
+                  }return null;
+                  },
+                  keyboardType: TextInputType.phone,
+                  obscureText: true,
+                  decoration: InputDecoration(
+                    icon: Icon(Icons.keyboard),
+                    labelText: 'RepeatPassword',
+                    border: OutlineInputBorder(
+                      borderRadius: BorderRadius.all(
+                        Radius.circular(5.0),
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(
+                height: 20,
+              ),
+              Container(height: 50,
+                child: TextFormField(
+                  focusNode: _codeFocusNode,
+                  controller: _codeController,
+                  decoration: InputDecoration(
+                    icon: Icon(Icons.vpn_key),
+                    labelText: 'Code(optional)',
+                    border: OutlineInputBorder(
+                      borderRadius: BorderRadius.all(
+                        Radius.circular(5.0),
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(
+                height: 25,
+              ),
+              Provider<CreateAccountO>.value(
+                // todo delete once goran creates observable
+                value: CreateAccountO(),
+                child: Consumer<CreateAccountO>(
+                  builder: (context, o, _) => Container(
+                    width: 180,
+                    height: 50,
+                    child: RaisedButton(
+                      color: Colors.pink,
```

Comment:
put the color again inside theme. If it is primary color, then pink will be automatically for raised button